cle/backends/pe/pe.py

```diff
@@ -62,7 +64,7 @@ def __init__(self, *args, **kwargs):
         self._symbol_cache = self._exports # same thing
         self._handle_imports()
         self._handle_exports()
-        self._handle_relocs()
+        self.__register_relocs()
```

Comment:
If I recall correctly, the elf method was double-underscored because it should NEVER be called outside of the context it is originally used, and attempting to do so will fuck everything up